chat-plugins/trivia.js

```diff
@@ -408,7 +389,14 @@ var Trivia = (function () {
 
 	return {
```

Comment:
also: why do you need a getInstance method at all? You could just return new Trivia().